### PR TITLE
ksw/set default title to filename

### PR DESCRIPTION
**Pull request by kswang1029**
**Description**
This is to address #2168 by setting the default custom title to filename in the frame constructor and setting the right title for AST re-rendering when the visible and custom title toggles change their states. 

**Checklist**

For linked issues (if there are):
- [x] assignee and label added
- [x] ZenHub issue connection, board status, and estimate updated

For the pull request:
- [x] reviewers and assignee added
- [x] ZenHub estimate, milestone, and release (if needed) added
- [x] e2e test passing / ~corresponding fix added~
- [x] changelog updated / ~no changelog update needed~
- [x] ~protobuf updated to the latest dev commit~ / no protobuf update needed
- [x] `BackendService` unchanged / ~`BackendService` changed and corresponding ICD test fix added~

### CODE REVIEW

**Comment by kswang1029** [comment]
@YuHsuan-Hwang thanks for the hints. now it is in a good state for review.

File: src/stores/Frame/FrameStore.ts [hunk 1]
```diff
@@ -1180,7 +1180,7 @@ export class FrameStore {
         this.moving = false;
         this.zooming = false;
         this.colorbarLabelCustomText = this.requiredUnit === undefined || !this.requiredUnit.length ? "arbitrary units" : this.requiredUnit;
-        this.titleCustomText = "";
+        this.titleCustomText = frameInfo.fileInfo.name;
```
Comment (YuHsuan-Hwang) [inline src/stores/Frame/FrameStore.ts]: add optional chaining: `frameInfo?.fileInfo?.name`. This will ensure unit tests don't break.

**Review by YuHsuan-Hwang — CHANGES_REQUESTED** [review]
Overall this looks good 👍

File: src/stores/Frame/FrameStore.ts (hunk 1 above)
Comment (kswang1029) [reply src/stores/Frame/FrameStore.ts]: fixed

**Review by YuHsuan-Hwang — APPROVED** [review]
I think we can add the change log item to `changed`.

**Review by acdo2002 — APPROVED** [review]
Looks good, setting the title of image viewer now is the filename itself. (dev is AST coordinate J2000 info instead of the filename itself)
Testing one and multi-images and export them as png, the title(s) is(are) displayed correctly.

**Review by crocka — APPROVED** [review]

**Pull request merged**